app/src/main/java/campus/tech/kakao/map/view/MainActivity.kt

```diff
@@ -99,55 +97,45 @@ class MainActivity : AppCompatActivity() {
                 setCameraPosition()
                 setMarker()
                 setBottomSheet()
+
             }
         })
     }
 
     private fun setCameraPosition() {
-        val sharedPreferences = getSharedPreferences("PlacePreferences", Context.MODE_PRIVATE)
-        val longitude =
-            sharedPreferences.getString(EXTRA_PLACE_LONGITUDE, "127.108621")?.toDouble() ?: 0.0
-        val latitude =
-            sharedPreferences.getString(EXTRA_PLACE_LATITUDE, "37.402005")?.toDouble() ?: 0.0
-        Log.d("testt", longitude.toString())
-        val cameraUpdate = CameraUpdateFactory.newCenterPosition(LatLng.from(latitude, longitude))
-        Log.d("testt", longitude.toString())
-        kakaoMap.moveCamera(cameraUpdate)
+        // LiveData 관찰
+        viewModel.placeData.observe(this, Observer { placeData ->
+            val cameraUpdate = CameraUpdateFactory.newCenterPosition(LatLng.from(placeData.latitude, placeData.longitude))
+            kakaoMap.moveCamera(cameraUpdate)
+        })
     }
 
     private fun setMarker() {
-        val sharedPreferences = getSharedPreferences("PlacePreferences", Context.MODE_PRIVATE)
-        val longitude =
-            sharedPreferences.getString(EXTRA_PLACE_LONGITUDE, "127.108621")?.toDouble() ?: 0.0
-        val latitude =
-            sharedPreferences.getString(EXTRA_PLACE_LATITUDE, "37.402005")?.toDouble() ?: 0.0
-        val placeName = sharedPreferences.getString(EXTRA_PLACE_NAME, "Unkwon")
-        var styles = LabelStyles.from(
-            LabelStyle.from(R.drawable.marker_128).setZoomLevel(10)
-                .setTextStyles(LabelTextStyle.from(32, Color.parseColor("#000000")))
-        )
-
-        styles = kakaoMap.labelManager!!.addLabelStyles(styles!!)
-        kakaoMap.labelManager!!.layer!!.addLabel(
-            LabelOptions.from(LatLng.from(latitude, longitude))
-                .setStyles(styles)
-                .setTexts(placeName)
-        )
+
+        // LiveData 관찰
+        viewModel.placeData.observe(this, Observer { placeData ->
+            val styles = LabelStyles.from(
+                LabelStyle.from(R.drawable.marker_128).setZoomLevel(10)
+                    .setTextStyles(LabelTextStyle.from(32, Color.parseColor("#000000")))
+            ).let {
+                kakaoMap.labelManager!!.addLabelStyles(it)
+            }
+
+            kakaoMap.labelManager!!.layer!!.addLabel(
```

Comment:
이전 피드백에서도 말씀드렸지만 여기선 labelManager가 non-null임을 100% 확신할 수 없기 때문에 분명 특정 상황에서는 앱크래시가 발생할 수 있습니다. safe call(?)을 사용해주시면 좋겠습니다.
```suggestion
                kakaoMap.labelManager?.addLabelStyles(it)
            }

            kakaoMap.labelManager?.layer?.addLabel(
```